.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,17 @@
-## This PR is a:
+## Purpose of this PR (required)
 
-- [ ] New topic
-- [ ] Content update
-- [ ] Content fix or rewrite
-- [ ] Bug fix or improvement
+Describe the goal and the types of changes this PR covers.
 
-## Summary
+## Affected URLS (required)
 
-When this pull request is merged, it will...
-
-<!-- (REQUIRED) What does this PR change? -->
-
-## Additional information
-
-List all affected URLs 
+List all files that this PR changes:
```

Comment:
The list of files is always available in the GitHub UI at **Files changed** tab

Reply:
This is nice to have up front for editors, groomers...

Reply:
Short list of the files is also available at **Jump to**  dropdown menu that can be used by editors and groomers.
This list can be very big, and can change during a PR history. Duplicating and maintaining it manually can be confusing for contributors. 
![Screen Shot 2019-04-05 at 14 18 48](https://user-images.githubusercontent.com/12731225/55651361-f998a500-57ad-11e9-9695-be923772ed0a.png)